### ISSUE TITLE
LibGuides: Interactive element does not meet minimum size nor spacing

### ISSUE THREAD

**Issue by JessWaggs** [UCSCLibrary/SiteImprove#126]
- [ ] List item links are too close together (Our new design system calls for more space between list items, will this solve the issue?)
- [ ] Links in footer are too close together (Does our new footer solve this issue?)

<img width="313" height="532" alt="Image" src="https://github.com/user-attachments/assets/451f10dd-26bf-433f-8a45-24b3b420a966" />
<img width="989" height="134" alt="Image" src="https://github.com/user-attachments/assets/854df589-5a24-42f4-9670-3da240b8362e" />

**Comment by JessWaggs** [comment]
It could be worthwhile to consider the AAA requirements for future-proofing, which are 44x44 pixels as the minimum size for interactive elements.
